clis/tcli/contrib/example_vpn_disable_linux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This is a simple example script on how to use DTCLI to turn off VPN and clean up routing.
+
+# NOTE: This is meant to be run from the libtelio directory.
+# And uses the debug build so that you could iterate quicker builds when debugging.
+
+DTCLI_PATH="target/debug/dtcli"
+
+# You may need to change these to your default values.
+TUNNEL_NAME="tun10"
```

Comment:
Can you add this as a parameter to the script ? If parameter not given, defaults to "tun10".

Reply:
Done